sandbox/src/mocks/retrieveReferralRequest/responses/cancelled.json

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
Hmm doesn't look like this file is currently being referenced anywhere, I'll add it as an example response both in the documentation and as part of the sandbox app. Other examples will need to be updated yep but as you mention this will be covered by other tickets linked to the individual defining state stories.